src/qt/bitcoingui.cpp

```diff
@@ -488,8 +488,11 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
             // Disable context menu on tray icon
             trayIconMenu->clear();
         }
-        // Propagate cleared model to child objects
+        // Propagate cleared model to RPC console
         rpcConsole->setClientModel(nullptr);
+
+        // Once RPC console is cleared, propagate to other child objects
+        this->clientModel = 0;
```

Comment:
```
this->clientModel = nullptr;
```

Reply:
Done. Changed to nullptr.
I had it set to 0 before because it is the current behavior in master. `BitcoinApplication::requestShutdown()` in bitcoin.cpp invokes `window->setClientModel(0)`